packages/moonstone/Input/InputSpotlightDecorator.js

```diff
@@ -27,6 +27,24 @@ const handleKeyDown = handle(
 	call('onKeyDown')
 );
 
+
+/**
+ * Default config for [InputSpotlightDecorator]{@link moonstone/Input.InputSpotlightDecorator}
+ *
+ * @memberof moonstone/Input/InputSpotlightDecorator.InputSpotlightDecorator
+ * @hocconfig
+ */
+const defaultConfig = {
+	/**
+	 * Suppress the pointer lock behavior of moonstone input
+	 *
+	 * @type {Boolean}
+	 * @default false
+	 * @memberof moonstone/Input/InputSpotlightDecorator.InputSpotlightDecorator.defaultConfig
+	*/
+	noLockPointer: false
```

Comment:
@ybsung, @jeongran1230 - I changed this name to `noLockPointer` to follow our usual convention for disabling default features. See `noAutoFocus` in `SpotlightRootDecorator` for some precedent on this. I know there's some dependent work on this so let me know if this change is a problem and we can adapt. It's not critical for a private API like this but want to strive for consistency when possible.

Reply:
@ryanjduffy - Okey, I updated the title of this PR.